File: primes/lua/primes.lua
#!/usr/bin/lua

for i=1,50000 do
	local comp=false
	for j=2,i-1 do
		local f=i
		local rf=f/j
		local r=math.floor(i/j)
		local ri=r
		if ri==rf then
			comp=true
			break
		end
	end
	if not comp then io.write(i,' -- lumple\n') end
end

File: primes/python/primes.py
#!/usr/bin/python

def go():
	for i in range(50000):
		comp=0
		for j in range (2,i):
			rf=float(i)/j
			ri=int(rf)
			if rf==ri:
				comp=1
				break

		if comp==0:
			print(str(i)+" -- simple")

go()
